### ISSUE TITLE
Questions about the global test accuracy

### ISSUE THREAD

**Issue by gtwilson8921** [zerocpp/fedhkd#2]
Could you kindly explain why the global test accuracy of the FedMD consistently hovers around 10%? This seems to deviate from the results reported in the paper. Is there an issue that needs to be addressed?

**Comment by gtwilson8921** [comment]
I understand the issue that has arisen. 'Global' and 'local' are two different training modes. In the 'global' mode, the server needs to transmit parameters to the clients, which means that on the server side, we need to include 'self.LocalModels[idx].load_model(self.global_model.state_dict())' in the code. Additionally, 'self.global_model.load_state_dict(global_weights)' should be added to obtain training results using 'self.global_test_accuracy()'. If you only wish to obtain local accuracy, there is no need to transmit model parameters.

**Comment by lunan0320** [comment]
> I understand the issue that has arisen. 'Global' and 'local' are two different training modes. In the 'global' mode, the server needs to transmit parameters to the clients, which means that on the server side, we need to include 'self.LocalModels[idx].load_model(self.global_model.state_dict())' in the code. Additionally, 'self.global_model.load_state_dict(global_weights)' should be added to obtain training results using 'self.global_test_accuracy()'. If you only wish to obtain local accuracy, there is no need to transmit model parameters.

Thanks for your issue, I suffered the same question with you. 
In the origin code, 'if self.args.upload_model == True:' is hidden and there is no hyper-parameter named 'upload_model' which should be added by us.  
Also,  'self.global_model.load_state_dict(global_weights)' is needed to take ahead of global_test.
